clients/src/test/java/org/apache/kafka/common/record/DefaultRecordBatchTest.java

```diff
@@ -153,6 +154,67 @@ public void testInvalidRecordSize() {
     }
 
     @Test(expected = InvalidRecordException.class)
+    public void testInvalidRecordSizeTooLow() {
```

Comment:
Thanks for spotting! Before refactoring that test was covering different case. Now the cases of invalid record count are being covered by the relocated `testInvalidRecordCount*` tests.